desktop/apps/auction/client/commercial_filter.js

```diff
@@ -1,3 +1,4 @@
+import analyticsMiddleware from './analytics'
```

Comment:
Maybe rename the file `analyticsMiddleware` so that we can better understand whats going on and keep parity?